build.gradle

```diff
@@ -9,8 +9,10 @@ buildscript {
     }
 }
 
+
+
+
```

Comment:
Don't add empty lines like this, please. Standard formatting would allow up to one empty line between commands, zero at the end of blocks, mandate one empty line between functions, etc.

```suggestion
```